web/public/javascripts/init.js

```diff
@@ -3,7 +3,7 @@ let app = Elm.Main.embed(node);
 
 let mymap;
 let markers = {};
-let polylines = [];
+let polyline;
```

Comment:
null type, it doesn't actually need to be a list since the polyline returned includes the previous polyline